framework/src/main/java/org/checkerframework/framework/stub/StubTypes.java

```diff
@@ -59,19 +59,33 @@
     /**
      * Mapping from fully-qualified class name to corresponding JDK stub file from the file system.
      */
-    private final Map<String, Path> jdk11StubFiles = new HashMap<>();
+    private final Map<String, Path> jdkStubFiles = new HashMap<>();
 
     /**
      * Mapping from fully-qualified class name to corresponding JDK stub files from the checker.jar.
      */
-    private final Map<String, String> jdk11StubFilesJar = new HashMap<>();
+    private final Map<String, String> jdkStubFilesJar = new HashMap<>();
+
+    /** Which version number of the annotated JDK should be used? */
+    private final String annotatedJdkVersion;
+
+    /** Should the JDK be parsed? */
+    private final boolean shouldParseJdk;
 
     /** Creates a stub type. */
     public StubTypes(AnnotatedTypeFactory factory) {
         this.factory = factory;
         this.typesFromStubFiles = new HashMap<>();
         this.declAnnosFromStubFiles = new HashMap<>();
         this.parsing = false;
+        String release = PluginUtil.releaseValue(factory.getProcessingEnv());
```

Comment:
Instead of returning a `@Nullable` string here and handling the fall-back at the call-site, how about moving that into `PluginUtil.releaseValue` and have that return an int?

Reply:
If I make that change, the method won't return the value of `release`, so I'll need to rename it. What should the new name be?

Reply:
How about `getReleaseVersion` to be symmetric with `getJreVersion`?

Reply:
We discussed this and decided to keep the current name.

Reply:
Although something like `getReleaseValue` would be probably closer to the naming of other methods in `PluginUtil`...

Reply:
Good point, I make this change.